SocialCareCaseViewerApi.Tests/V1/Domain/VisitTests.cs

```diff
@@ -1,54 +0,0 @@
-using System;
-using NUnit.Framework;
-using SocialCareCaseViewerApi.V1.Domain;
-
-namespace SocialCareCaseViewerApi.Tests.V1.Domain
-{
-    public class VisitTests
```

Comment:
Should we stop writing these sort of tests altogether? As long as there is proper test coverage these are probably pointless. We have discussed checking for properties and their types in objects, but that might not provide any additional value.

Reply:
I don't believe there to be any value in these tests.

I don't think there is any value in testing for the properties either.